From 7bee69666cd489546c20e4bd02215d1f4e3642f7 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Fri, 6 Nov 2020 19:41:48 +0800
Subject: [PATCH] mmc: sdhci-of-dwcmshc: Fix wrong definition of
 DWCMSHC_EMMC_DLL_STATUS0

Acorrding to the latest TRM, the offset of DWCMSHC_EMMC_DLL_STATUS0
is 0x840. So fix it.

Change-Id: I85e4b04b0d68654c988df26f29f03cfe4301831b
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 drivers/mmc/host/sdhci-of-dwcmshc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/host/sdhci-of-dwcmshc.c b/drivers/mmc/host/sdhci-of-dwcmshc.c
index bcf33a63d353..fe23c77c056c 100644
--- a/drivers/mmc/host/sdhci-of-dwcmshc.c
+++ b/drivers/mmc/host/sdhci-of-dwcmshc.c
@@ -31,7 +31,7 @@
 #define DWCMSHC_EMMC_DLL_TXCLK		0x804
 #define DWCMSHC_EMMC_DLL_RXCLK		0x808
 #define DWCMSHC_EMMC_DLL_STRBIN		0x80c
-#define DWCMSHC_EMMC_DLL_STATUS0	0x820
+#define DWCMSHC_EMMC_DLL_STATUS0	0x840
 #define DWCMSHC_EMMC_DLL_START		BIT(0)
 #define DWCMSHC_EMMC_DLL_RXCLK_SRCSEL	29
 #define DWCMSHC_EMMC_DLL_START_POINT	16
-- 
2.35.3

